index.js

```diff
@@ -57,6 +64,35 @@ module.exports.mime = mime
 var listenerCount = EventEmitter.listenerCount
   || function(emitter, type){ return emitter.listeners(type).length; };
 
+
+function responseStatus(res, code, msg)
+{
+  if(msg == null)
```

Comment:
Please match existing styles, for example the bracket style really stands out as misplaced.